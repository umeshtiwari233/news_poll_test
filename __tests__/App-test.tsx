/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

// it('renders correctly', () => {
//   renderer.create(<App />);
// });

test('snapshot testing',()=>{
  const snap=renderer.create(
    <App/>
  ).toJSON();
  expect(snap).toMatchSnapshot();
})
