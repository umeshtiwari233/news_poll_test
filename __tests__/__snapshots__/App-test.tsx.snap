// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<View>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "rgb(238, 232, 244)",
        "borderRadius": 28,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="search-bar-container-outer-layer"
  >
    <View
      collapsable={false}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "rgb(238, 232, 244)",
          "borderRadius": 28,
          "flex": undefined,
          "flexDirection": "row",
          "shadowColor": "#000",
          "shadowOffset": {
            "height": 0,
            "width": 0,
          },
          "shadowOpacity": 0,
          "shadowRadius": 0,
        }
      }
      testID="search-bar-container"
    >
      <View
        collapsable={false}
        style={
          {
            "backgroundColor": "transparent",
            "borderRadius": 20,
            "height": 40,
            "margin": 6,
            "shadowColor": "#000",
            "shadowOffset": {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0,
            "shadowRadius": 0,
            "width": 40,
          }
        }
        testID="icon-button-container-outer-layer"
      >
        <View
          collapsable={false}
          style={
            {
              "backgroundColor": "transparent",
              "borderColor": "rgba(121, 116, 126, 1)",
              "borderRadius": 20,
              "borderWidth": 0,
              "elevation": 0,
              "flex": 1,
              "overflow": "hidden",
              "shadowColor": "#000",
              "shadowOffset": {
                "height": 0,
                "width": 0,
              },
              "shadowOpacity": 0,
              "shadowRadius": 0,
            }
          }
          testID="icon-button-container"
        >
          <View
            accessibilityComponentType="button"
            accessibilityLabel="search"
            accessibilityRole="button"
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": true,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityTraits="button"
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            centered={true}
            collapsable={false}
            focusable={true}
            hitSlop={
              {
                "bottom": 6,
                "left": 6,
                "right": 6,
                "top": 6,
              }
            }
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              [
                {
                  "overflow": "hidden",
                },
                [
                  {
                    "alignItems": "center",
                    "flexGrow": 1,
                    "justifyContent": "center",
                  },
                  {
                    "borderRadius": 20,
                  },
                ],
              ]
            }
            testID="icon-button"
          >
            <Text
              accessibilityElementsHidden={true}
              importantForAccessibility="no-hide-descendants"
              pointerEvents="none"
              selectable={false}
              style={
                [
                  {
                    "backgroundColor": "transparent",
                  },
                  {
                    "color": "rgba(73, 69, 79, 1)",
                    "fontSize": 24,
                  },
                ]
              }
            >
              □
            </Text>
          </View>
        </View>
      </View>
      <TextInput
        accessibilityRole="search"
        keyboardAppearance="light"
        onChangeText={[Function]}
        placeholder="Search"
        placeholderTextColor="rgba(28, 27, 31, 1)"
        returnKeyType="search"
        selectionColor="rgba(103, 80, 164, 1)"
        style={
          [
            {
              "alignSelf": "stretch",
              "flex": 1,
              "fontSize": 18,
              "minWidth": 0,
              "paddingLeft": 8,
              "textAlign": "left",
            },
            {
              "color": "rgba(73, 69, 79, 1)",
              "fontFamily": "System",
              "fontSize": 16,
              "fontWeight": "400",
              "letterSpacing": 0.15,
              "lineHeight": 0,
            },
            {
              "minHeight": 56,
              "paddingLeft": 0,
            },
            undefined,
          ]
        }
        testID="search-bar"
        underlineColorAndroid="transparent"
        value=""
      />
      <View
        pointerEvents="none"
        style={
          [
            {
              "marginLeft": 16,
              "position": "absolute",
              "right": 0,
            },
            false,
          ]
        }
        testID="search-bar-icon-wrapper"
      >
        <View
          collapsable={false}
          style={
            {
              "backgroundColor": "transparent",
              "borderRadius": 20,
              "height": 40,
              "margin": 6,
              "shadowColor": "#000",
              "shadowOffset": {
                "height": 0,
                "width": 0,
              },
              "shadowOpacity": 0,
              "shadowRadius": 0,
              "width": 40,
            }
          }
          testID="icon-button-container-outer-layer"
        >
          <View
            collapsable={false}
            style={
              {
                "backgroundColor": "transparent",
                "borderColor": "rgba(121, 116, 126, 1)",
                "borderRadius": 20,
                "borderWidth": 0,
                "elevation": 0,
                "flex": 1,
                "overflow": "hidden",
                "shadowColor": "#000",
                "shadowOffset": {
                  "height": 0,
                  "width": 0,
                },
                "shadowOpacity": 0,
                "shadowRadius": 0,
              }
            }
            testID="icon-button-container"
          >
            <View
              accessibilityComponentType="button"
              accessibilityLabel="clear"
              accessibilityRole="button"
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": false,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityTraits="button"
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              centered={true}
              collapsable={false}
              focusable={true}
              hitSlop={
                {
                  "bottom": 6,
                  "left": 6,
                  "right": 6,
                  "top": 6,
                }
              }
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                [
                  {
                    "overflow": "hidden",
                  },
                  [
                    {
                      "alignItems": "center",
                      "flexGrow": 1,
                      "justifyContent": "center",
                    },
                    {
                      "borderRadius": 20,
                    },
                  ],
                ]
              }
              testID="icon-button"
            >
              <Text
                accessibilityElementsHidden={true}
                importantForAccessibility="no-hide-descendants"
                pointerEvents="none"
                selectable={false}
                style={
                  [
                    {
                      "backgroundColor": "transparent",
                    },
                    {
                      "color": "rgba(255, 255, 255, 0)",
                      "fontSize": 24,
                    },
                  ]
                }
              >
                □
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
  <RCTScrollView
    data={[]}
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onEndReached={[Function]}
    onEndReachedThreshold={0}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={[]}
    testID="flatlist1"
    viewabilityConfigCallbackPairs={[]}
  >
    <View />
  </RCTScrollView>
</View>
`;
